docker/webui/run_openqa.sh

```diff
@@ -7,9 +7,36 @@ else
   /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
 fi
 
+function scheduler(){
+  su -u /usr/share/openqa/script/openqa-scheduler-daemon
+}
+
+function websockets(){
+  su geekotest -c /usr/share/openqa/script/openqa-websockets-daemon
+}
+
+function gru() {
+  su geekotest -c /usr/share/openqa/script/openqa-gru -m production run
+}
+
+function webui() {
+  su geekotest -c /usr/share/openqa/script/openqa-webui-daemon &
+  apache2ctl start
+  while :; do sleep 100; done;;
+}
+
+
 # run services
-start_daemon -u geekotest /usr/share/openqa/script/openqa-scheduler &
-start_daemon -u geekotest /usr/share/openqa/script/openqa-websockets &
-start_daemon -u geekotest /usr/share/openqa/script/openqa gru -m production run &
-apache2ctl start
-start_daemon -u geekotest /usr/share/openqa/script/openqa prefork -m production --proxy
+case "$MODE" in
+  scheduler ) scheduler;;
+  websockets ) websockets;;
+  gru ) gru;;
+  backend ) scheduler &
+            websockets &
+            gru;;
```

Comment:
What is the benefit of running multiple daemons in one container? This way the log of the services is combined, right? (Which makes it harder to read.) What if I just want to restart one of the daemons?

Reply:
Is a good point. maybe this option could be ommited.

Reply:
Still, I consider this a valid use case and this is how it was in before. I see no need to remove that right now